docs/INSTALLATION.md

```diff
@@ -1,15 +1,7 @@
 # Installation
 
-<!-- TODO: write document
+This track uses the [nightly Arturo releases][nightly]. The `full` build is required for package support and is currently supported for only Linux and Windows platforms. An [official Docker image][docker-image] is also available.
```

Comment:
This is pretty light on install details. Is it worth adding details? Are any additional packages needed, eg for the test framework?

Reply:
We dropped the info earlier but Arturo nightly imports the unitt package, installing it if needed. So there’s not much to it. No other packages should be needed for Exercism use since Arturo includes a lot in its standard library.

I should make crystal clear the nightly version of Arturo is needed. The release version is pretty stale and won’t have any package manager support.

Reply:
I agree with Isaac here. Can you at least have a list of steps to download and install such that the user can run exercise tests?